packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -98,24 +96,33 @@ export const makeContract = harden((zoe, terms) => {
   const makeInvite = () => {
     const seat = harden({
       addOrder: () => {
-        // Is it a valid sell offer?
-        if (hasValidPayoutRules(['offerAtMost', 'wantAtLeast'], inviteHandle)) {
+        const {
+          offerRules: { offer, want },
+        } = zoe.getOffer(inviteHandle);
+        const [offerRoleName] = Object.getOwnPropertyNames(offer);
+        const [wantRoleName] = Object.getOwnPropertyNames(want);
+        const expected = harden({
+          offer: [offerRoleName],
+          want: [wantRoleName],
+        });
+        rejectIfNotOfferRules(inviteHandle, expected);
```

Comment:
I think this check isn't actually checking anything. This checks that the roleNames that we got from the offerRules match the roleNames in the offerRules.

Reply:
We can add a method to check the offerRules against expected, but not reject. That seems like the best way to do this.

Reply:
I added a method. Let me know if it helps or needs adjustment! https://github.com/Agoric/agoric-sdk/pull/532/commits/3934c71ee4c1545db8e6ea12e33dc6c08ea5bdc6